### ISSUE TITLE
QoL improvements

### ISSUE THREAD

**Issue by rogerro007** [yaser01/mkv-muxing-batch-gui#41]
1. Refresh when new files are added or deleted to the folders with videos/attachments (could be auto or button to refresh)
2. Subtitle set as deafult and forced - allow for deafault to be check/uncheck

**Comment by yaser01** [comment]
Nice ideas, will think about them in the next update :)
For the second one you mean to have set default/forced auto checked ?

**Comment by rogerro007** [comment]
yeah, now every time i launch program I must remember to check those option
